### ISSUE TITLE
Error receive a message with media

### ISSUE THREAD

**Issue by ikkehman** [open-wa/wa-automate-nodejs#1339]
When receive/get a messages with media like images for make sticker, i got some error

> [ERROR] Error: message is missing critical data needed to download the file.
    at /root/Wibot-v3/node_modules/@open-wa/wa-decrypt/dist/decrypt.js:83:27
    at step (/root/Wibot-v3/node_modules/@open-wa/wa-decrypt/dist/decrypt.js:44:23)
    at Object.next (/root/Wibot-v3/node_modules/@open-wa/wa-decrypt/dist/decrypt.js:25:53)
    at /root/Wibot-v3/node_modules/@open-wa/wa-decrypt/dist/decrypt.js:19:71
    at new Promise (<anonymous>)
    at __awaiter (/root/Wibot-v3/node_modules/@open-wa/wa-decrypt/dist/decrypt.js:15:12)
    at exports.decryptMedia (/root/Wibot-v3/node_modules/@open-wa/wa-decrypt/dist/decrypt.js:67:71)
    at msgHandler (/root/Wibot-v3/msgHandler.js:81:53)
    at process._tickCallback (internal/process/next_tick.js:68:7)

it's worked as well before and i do nothing with my script.

**Comment by odhier** [comment]
same issue,

when i see decrypt.js 
`
>...
> if (!message.clientUrl)
>                     throw new Error('message is missing critical data needed to download the file.');
> ...

`

its block that throw the error, so i console.log **message** and there is no _clientUrl_ property. is it update from whatsapp?

> {
>   type: 'video',
>   deprecatedMms3Url: 'https://mmg.whatsapp.net/d/f/Ak6evbFcPwhx2mnavRrW_1A2DmhYmiMgI3t16EgZ7Y5e.enc',
>   directPath: '/v/t62.7161-24/40252785_1359305954430288_929662952623127763_n.enc?oh=1150cc9902b51a71cb340a403a2091fa&oe=604EE8F2',
>   mimetype: 'video/mp4',
>   filehash: 'VPoPZOLA4QY1IkCoRiUTsXRQWju8806c9UzSqWh2X+Q=',
>   encFilehash: 'NAD3SE7s+BfesfCTlsnJUnO+17A4qlGPgZJTRo+QUqE=',
>   size: 209080,
>   height: 288,
>   width: 268,
>   duration: '4',
>   mediaKey: 'VnlzS7AWV7echryvy15ydIhLClUVb3s5z/6CPYgWfRQ=',
>   mediaKeyTimestamp: 1613039187,
>   body: '/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEABsbGxscGx4hIR4qLSgtKj04MzM4PV1CR0JHQl2NWGdYWGdYjX2Xe3N7l33gsJycsOD/2c7Z//////////////8BGxsbGxwbHiEhHiotKC0qPTgzMzg9XUJHQkdCXY1YZ1hYZ1iNfZd7c3uXfeCwnJyw4P/Zztn////////////////CABEIAEgAQwMBIgACEQEDEQH/xAAwAAACAwEBAAAAAAAAAAAAAAAABQMEBgIBAQADAQEAAAAAAAAAAAAAAAAAAQIDBP/aAAwDAQACEAMQAAAAZgahWmqpx8NEstv2pbVIAwAapnkUWxq2aiuk6zekcAFSHs4ZmhLDnT9RoMm3M9zb8lqBZNW8Qa81pW6oYb6TJbbPiqPo+W5yqU+KAbctiiGO2sAmqXAU1ICP/8QAJxAAAgICAQIGAwEBAAAAAAAAAQIAAwQREhMhBRAiMUFhFCBRcST/2gAIAQEAAT8A87sg1uK0Xk5jpnJ65VnHlweI4cb/AFscIjMfgTDKj1nuzmW5CodGZ9akdZJ4Y/MN+ucdY7SilFRXAJIE0to2yCZSA1lNdjKFGNeg/o7/AK55AxzMW3qUBWbjMdKe4WzZniJZDUFMBPX2WJO4p2o8wCfaZ+R1D0x8GJyJRdzFq6CNa3uZlZBdj3gYr6iZgZJuQg+6+QBJAjjhWR9SwBOfySYvcqRFHLE2R34x/dv9jbOhPC6LAjPrsfIca15MZ+dUSQRM/ERd2L7GVb5qPuBTwH+S9f8AosBGu8wsNcljymMxqr6R+IXBMy2D0tobilmcDuJktrGZN7ImHr8ivt5eIUWX5KBU7THNONXx169QOz7JI0Z+ZjJ6S0sJVORfj9Q9TuysDMMG4Wc/5KudOWqN8NPcQKu/uZVaSlVAGzL8ew2uU7jc/8QAHBEAAwADAQEBAAAAAAAAAAAAAAECEBExIQMy/9oACAECAQE/ABnBZfcTjZv0ZOI+c0m6KnVD9RKwvzsvqGLmP//EABwRAAICAwEBAAAAAAAAAAAAAAACAREQICFBMf/aAAgBAwEBPwAiLK580U9J0vLM18Em1FJynpGf/9k=',
>   interactiveAnnotations: [],
>   isViewOnce: false
> }
>

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
There are some issues with the build action.

Please upgrade the decryption library manually. In your project folder do this:

```bash
> npm i @open-wa/wa-decrypt@3.0.1
```

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.4.1](https://github.com/open-wa/wa-automate-nodejs/compare/3.4.0...3.4.1) (2021-02-12)

- build(deps): bump ts-json-schema-generator from 0.83.2 to 0.84.0 [`#1324`](https://github.com/open-wa/wa-automate-nodejs/pull/1324)
- build(deps): bump puppeteer from 7.0.0 to 7.0.4 [`#1331`](https://github.com/open-wa/wa-automate-nodejs/pull/1331)
- build(deps): bump swagger-ui-dist from 3.42.0 to 3.43.0 [`#1341`](https://github.com/open-wa/wa-automate-nodejs/pull/1341)
- build(deps-dev): bump @types/node from 14.14.25 to 14.14.26 [`#1342`](https://github.com/open-wa/wa-automate-nodejs/pull/1342)
- Release 3.4.1 [`eedc286`](https://github.com/open-wa/wa-automate-nodejs/commit/eedc28667bc19c39a707712e3176d849518bd988)
- deps: updated wa-decrypt to 3.0.1 [`e443571`](https://github.com/open-wa/wa-automate-nodejs/commit/e44357188af00356f95505fc11a6aae3a13e88d4)
- deps: revert to puppeteer 7.0.0 [`50c7437`](https://github.com/open-wa/wa-automate-nodejs/commit/50c74375889f6c8b042531282e4fdff0b28e06b8)
- fix: export id types [`9327db0`](https://github.com/open-wa/wa-automate-nodejs/commit/9327db0c1629fbe5e7986aadb5cf6407dd7598af)
- fix: export media types [`f5ce50f`](https://github.com/open-wa/wa-automate-nodejs/commit/f5ce50f5857a911a3289c75dc09e9b5520278165)
- patch: fixed a few bugs due to wa update [`9a1d3c3`](https://github.com/open-wa/wa-automate-nodejs/commit/9a1d3c38106be15db8fd9b55f50de8c0edc75104)
- patch: decryption error fixed [`7ccf5f2`](https://github.com/open-wa/wa-automate-nodejs/commit/7ccf5f21e4d2abdbd25836eeb67ae759a68f4e03)
